### ISSUE TITLE
Update question title for senior housing question on eligibility questionnaire

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#839]
We would like to update the question title for the senior housing question on the eligibility questionnaire. The question currently reads 'How old are you?'. We're currently awaiting content on exactly what the new copy should be, so I'll mark this issue as blocked for now.

**Comment by jnowell** [comment]
The new question is 'How old are the people who will live with you, including yourself?'
